### ISSUE TITLE
Fix typo in oscardemo.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8482]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8482|
|Type|defect|
|Created|2016-06-17 02:55:43Z|
|Branch|https://github.com/twisted/twisted/tree/8482-rodrigc-typo|

Fix typo in oscardemo.py, introduced by [ticket:8346]

<details><summary>Searchable metadata</summary>

```
trac-id__8482 8482
type__defect defect
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8482_rodrigc_typo 8482-rodrigc-typo
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1466132143986357 1466132143986357
changetime__1466719998199885 1466719998199885
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/194

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Look good. Thanks! Please merge.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset f7f31618b24c3ebfc7bd58d2917f07fea8a27da8
```
#!CommitTicketReference repository="" revision="f7f31618b24c3ebfc7bd58d2917f07fea8a27da8"
Merge 8482-rodrigc-typo: Fix typo in oscardemo.py

Author: rodrigc
Reviewed by: adiroiban, alex
Fixes: #8482
```
